### DISCUSSION TITLE
Accidentally turned on some strange feature that messes up the listing. What is it and how to turn it off?

### DISCUSSION THREAD

**Discussion by hippietrail** [NationalSecurityAgency/ghidra#6919] [General]
I'm editing a binary file, just setting parts to arrays in bytes of various lengths and setting comments. And sometimes adjusting column widths to make sure the number of bytes field is wide enough.

At some point I either hit a key by mistake or took my finger off the mouse button during a context menu, or something similar. Then all these red/brown numbers `76` and `77` appeared between the listing lines:
<img width="508" alt="Screenshot 2024-09-16 at 7 06 51 PM" src="https://github.com/user-attachments/assets/a46d7819-ea9f-48a5-8f6a-5fbdcbe6b958">

I immediately hit undo, which had no effect. Hovering the mouse pointer over the numbers doesn't tell me anything. I tried quitting Ghidra and starting again and they're still there.

What are they and how can I make them go away?

**Comment by dragonmacher** [comment]
It looks like you added an optional field to the Listing.  You can disable that field from the Listing's toolbar.

Find the icon that looks like a keyboard with a down arrow, with the text "Edit the Listing Fields".  With that UI open, as you click in fields, they will be highlighted in that UI.  In the Listing, lick on the field you do not want (the "77" in this case).   Then, back in the header UI, right-click on the highlighted name and select either Disable Field or Remove Field.    Press the icon again in the toolbar to close that UI.

**Comment by hippietrail** [comment]
Thank you so much!
